src/main/java/frc/robot/Constants.java

```diff
@@ -35,44 +35,44 @@ public static final class DriveConstants {
 
         public static final int frontLeftDriveID = 1;
 		public static final int backLeftDriveID = 3;
-		public static final int frontRightDriveID = 7;
 		public static final int backRightDriveID = 5;
+		public static final int frontRightDriveID = 7;
 
-        public static final int frontLeftRotationMotorID = 2; 
-        public static final int frontRightRotationMotorID = 4; 
-        public static final int backLeftRotationMotorID = 8; 
-        public static final int backRightRotationMotorID = 6; 
+        public static final int frontLeftRotationMotorID = 2;
+        public static final int frontRightRotationMotorID = 4;
+        public static final int backRightRotationMotorID = 6;
+        public static final int backLeftRotationMotorID = 8;
 
         public static final int frontLeftRotationEncoderID = 1; 
         public static final int frontRightRotationEncoderID = 2; 
+        public static final int backRightRotationEncoderID = 3;
         public static final int backLeftRotationEncoderID = 4; 
-        public static final int backRightRotationEncoderID = 3; 
 
         public static final double frontLeftAngleOffset = 0.739378739760879;
         public static final double frontRightAngleOffset = 2.555611992617478; 
-        public static final double backLeftAngleOffset = 2.551010050253821;
-        public static final double backRightAngleOffset = 5.533068701903508;
+        public static final double backRightAngleOffset = 5.533068701903508; // TODO: Might have to switch these
```

Comment:
If you subtract PI here and invert the motor, that might be a good move (at least to be consistent with how we've done this in the past)

Reply:
Done I believe?